phpcs-ruleset.xml

```diff
@@ -9,6 +9,12 @@
     <exclude-pattern>*.min.js</exclude-pattern>
     <exclude-pattern>profiles/common/themes/ec_resp/scripts/ec.js</exclude-pattern>
     <exclude-pattern>profiles/common/themes/ec_resp/css/ec_resp.css</exclude-pattern>
+    <exclude-pattern>profiles/common/themes/europa/js/html5.js</exclude-pattern>
+    <exclude-pattern>profiles/common/themes/europa/js/modernizr.js</exclude-pattern>
+    <exclude-pattern>profiles/common/themes/europa/styleguide/template/custom/public/kss.css</exclude-pattern>
+    <exclude-pattern>profiles/common/themes/europa/styleguide/template/custom/public/less.js</exclude-pattern>
+    <exclude-pattern>profiles/common/themes/europa/styleguide/assets/public/kss.css</exclude-pattern>
+    <exclude-pattern>profiles/common/themes/europa/styleguide/assets/public/less.js</exclude-pattern>
 
```

Comment:
* Instead of adding these files to the list, they should simply be renamed to {{*.min.js}} and {{*.min.css}}, that can be done in scope of this ticket.
* It looks like {{modernizr}} is not used, we should make a ticket for this to remove it from the theme.
* The two files named {{less.js}} are empty and should be removed.